content/docs/command-reference/commit.md

```diff
@@ -17,12 +17,11 @@ positional arguments:
 
 ## Description
 
-The `dvc commit` command is useful for several scenarios, when data already
-tracked by DVC changes: when a [stage](/doc/command-reference/run) or
-[pipeline](/doc/command-reference/dag) is in development/experimentation; to
-force-update the `dvc.lock` or `.dvc` files without reproducing stages or
-pipelines; or to mark existing files/dirs as stage <abbr>outputs</abbr>. These
-scenarios are further detailed below.
+The `dvc commit` command is useful for several scenarios: when a
+[stage](/doc/command-reference/run) or [pipeline](/doc/command-reference/dag) is
+in development/experimentation; to force-update the `dvc.lock` or `.dvc` files
+without reproducing stages or pipelines; or when modifying stage
+<abbr>outputs</abbr> without DVC. These scenarios are further detailed below.
```

Comment:
`without DVC` - it won't be clear what does it mean. Better to specify _with_ (how) vs _without_ - it is more explicit.

Reply:
Working on this I realized there are errors in the cmd ref so I'm rewriting it again (based on previous text but much shorter) avoiding repeating implementation details... ⌛

Reply:
OK I corrected and simplified (also reorged the text) of all of this, included add/run/repo --no-commit options in ae902867. PTAL

Reply:
BTW please also check this core request I made: iterative/dvc/issues/5000